main.py

```diff
@@ -11,11 +11,8 @@ def main():
     "This is the main function. It all starts here."
 
     matchesDatabase = database.connect("noschemas")
-<<<<<<< HEAD
```

Comment:
I just picked the old ones to keep, I actually didn't know that any pushes to my fork will add to the pull request.